granary/bluesky.py

```diff
@@ -2257,14 +2272,20 @@ def upload_media(self, media):
 
       with util.requests_get(url, stream=True) as fetch:
         fetch.raise_for_status()
+        data = BytesIO(util.FileLimiter(fetch.raw, MAX_MEDIA_SIZE_BYTES).read())
+        content_type = fetch.headers['Content-Type']
```

Comment:
```suggestion
        content_type = fetch.headers.get('Content-Type', '')
```

...so that it doesn't raise `KeyError` when `Content-Type` is missing